test/models/game_test.rb

```diff
@@ -4,4 +4,308 @@ class GameTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+
+
+  #####################################################################################################
+	################################# POPULATE BLACK PIECES #############################################
+	#####################################################################################################
+
+
+  ##### TEST POPULATE 2 BLACK ROOKS ######
+
+	test "populate black rook - 1 of 2" do
+	  game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		expected = "Rook0"
+		rook = game.pieces.where(:pos_x => 0).where(:pos_y => 0).first
+		actual = rook.type.to_s + rook.color.to_s
+		assert_equal expected, actual
+	end
+
+	test "populate black rook - 2 of 2" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		expected = "Rook0"
+		rook = game.pieces.where(:pos_x => 7).where(:pos_y => 0).first
+		actual = rook.type.to_s + rook.color.to_s
+		assert_equal expected, actual
+	end
+
+
+  ##### TEST POPULATE 2 BLACK KNIGHTS ######
+
+	test "populate black knight - 1 of 2" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		expected = "Knight0"
+		knight = game.pieces.where(:pos_x => 1).where(:pos_y => 0).first
+		actual = knight.type.to_s + knight.color.to_s
+		assert_equal expected, actual
+	end
+
+	test "populate black knight - 2 of 2" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		expected = "Knight0"
+		knight = game.pieces.where(:pos_x => 6).where(:pos_y => 0).first
+		actual = knight.type.to_s + knight.color.to_s
+		assert_equal expected, actual
+	end
+
+
+  ##### TEST POPULATE 2 BLACK BISHOPS ######
+
+	test "populate black bishop - 1 of 2" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		expected = "Bishop0"
+		bishop = game.pieces.where(:pos_x => 2).where(:pos_y => 0).first
+		actual = bishop.type.to_s + bishop.color.to_s
+		assert_equal expected, actual
+	end
+
+	test "populate black bishop - 2 of 2" do
+		game = Game.create(:white_uid => 1, :black_uid => 1, :game_name => "New Game")
+		expected = "Bishop0"
```

Comment:
The only thing i'd say is that this is a little strange.  There is no reason you can assert 2 things in a test.  So for example you can have

assert_equal "Bishop", bishop.type.to_s
assert_equal 0, bishop.color